manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -165,7 +165,7 @@ spec:
               resourceFieldRef:
                 resource: limits.cpu
           - name: PLATFORM
-            value: "{{ .Values.global.platform }}"
+            value: "{{ .Values.platform }}"
```

Comment:
I think we need to temporarily support both values for backward compatibility?

Reply:
I was kind of expecting to just have the gateway chart use `.Values.platform` and the rest continue using `global.platform`. This is cnsistent with some other fields I think.

The key principal is there is no unity of values between charts. `Values.foo.bar` in chart 1 is unrelated to `Values.foo.bar` in chart 2. Anything treating them the same -- like IstioOperator -- is flawed.